utils/diagnostics/t/README.md

```diff
@@ -20,16 +22,20 @@ echo "this to the log: this test was run as user $USER" 1>&5
 
 ```
 
-As you can see, any output that is sent to FD5 will end up in the appropriate log file.
+As you can see, any output that is sent to FD5 will end up in the appropriate
+log file.
 
 ____
 
-When run, any scripts matching utils/diagnostics/t/*.t will be executed in lexical order.
+When run, any scripts matching utils/diagnostics/t/*.t will be executed in
+lexical order.
 
 - std in and std out are available to those scripts for user interaction.
 - fd5 will be redirected to a logfile named the same as the test script
 - all logfiles will be created in a tempdir created with `mktemp -dt lsmb-diag.XXXX`
 - a `.lock` file will be created within each tempdir and removed on exit from diagnose.sh
 - all non-locked tempdirs will be removed at the start of running `diagnose.sh`
-- `diagnose.sh` handles creation of the tempdir, and actual redirection of fd5 to the logfile
-- `diagnose.sh` creates a tarball of the contents of the tempdir before "normal" exit
+- `diagnose.sh` handles creation of the tempdir, and actual redirection of fd5
+to the logfile
+- `diagnose.sh` creates a tarball of the contents of the tempdir before
+"normal" exit
```

Comment:
In other files a `*` has been used for unordered lists. 
I'd suggest replacing the `-` with a `*` to conform.
I'm surprised that the Linter didn't report this

Reply:
Default check is set to consistent within a file. We can force it to `*`

Reply:
Well, the github markdown spec is to use an asterisk so we should require that

Reply:
Fixed